Rmarkdown/Multiple_Ascending_Dose_PD_ordinal.Rmd

```diff
@@ -205,9 +230,31 @@ gg <- gg + facet_grid(.~DAY_label)
 gg
 ```
 
+### Percent of subjects by response category against dose, faceted by day
+
+```{r, error = TRUE, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message = FALSE}
+data_to_plot <- pd_data %>% subset(PROFDAY %in% PD_PROFDAYS, )
+
+gg <- ggplot(data = data_to_plot, aes(x = factor(DOSE), fill = ORDINAL_LEVELS_high2low)) 
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + geom_bar(position = "fill") + scale_y_continuous(labels = scales::percent)
+gg <- gg + labs(x = dose_label, y = pd_response_label) + guides(fill = guide_legend(""))
+gg <- gg + scale_fill_brewer(palette = 6)
+gg <- gg + facet_grid(.~DAY_label)
+gg
+
+gg <- ggplot(data = data_to_plot, aes(x = DOSE, response = ORDINAL_LEVELS_high2low, color = ORDINAL_LEVELS_high2low)) 
+gg <- gg + xgx_annotate_status(status)
+gg <- gg + xgx_stat_ci(distribution = "ordinal", position = position_dodge(width = 50), alpha = 0.5) 
+gg <- gg + scale_y_continuous(labels = scales::percent)
+gg <- gg + labs(x = dose_label, y = pd_response_label) + guides(color = guide_legend(""))
+gg <- gg + facet_grid(.~DAY_label)
+gg
+```
+
 ### Ordinal response against dose, faceted by day
 
```

Comment:
Would it make sense to add in the ordinal regression curves from xgx_stat_smooth(method = "polr"), similar to how you have it for the PKPD ordinal data?